lib/proc/proc_tokenize.c

```diff
@@ -54,6 +54,29 @@ fin_tokens(grn_ctx *ctx, grn_obj *tokens)
   GRN_OBJ_FIN(ctx, tokens);
 }
 
+static void
+output_only_token_values(grn_ctx *ctx, grn_obj *tokens, grn_obj *lexicon)
```

Comment:
How about the function name `output_simple_tokens`?

Reply:
`output_tokens_simple()`?

Reply:
fix: [06a08ad](https://github.com/groonga/groonga/pull/2302/commits/06a08adda05355e58ec9030905e2406d63e94783)
Thank you so much.
I follow Groonga’s post-modifier naming convention like `output_columns_xx`.